xml/MAIN-SBP-Migrate-z-KVM.xml

```diff
@@ -454,7 +454,7 @@
         <para>For basic troubleshooting information review the section
                 <quote>Troubleshooting</quote> of the SUSE Linux Enterprise Server Administration
             guide at <link
-                xlink:href="https://www.suse.com/documentation/sles-12/book_sle_admin/data/part_trouble.html"
+                xlink:href="https://documentation.suse.com/sles/12-SP4/html/SLES-all/cha-p.html#sec-drucken-prob"
```

Comment:
...and here.

Reply:
See above regarding MAIN-SBP-Migrate-z-KVM.xml